src/SelectField/SelectField.js

```diff
@@ -228,6 +233,7 @@ class SelectField extends Component {
         underlineFocusStyle={underlineFocusStyle}
       >
         <DropDownMenu
+          {...dropDownMenuProps}
```

Comment:
Here we are 😄 . One last thing, it should be at the end so all properties can be overridden.